src/OrchardCore/OrchardCore.Admin.Abstractions/AdminPermissions.cs

```diff
@@ -0,0 +1,8 @@
+using OrchardCore.Security.Permissions;
+
+namespace OrchardCore.Admin;
+
+public class AdminPermissions
+{
+    public static readonly Permission AccessAdminPanel = new("AccessAdminPanel", "Access admin panel");
```

Comment:
Since it's not apparent from just by looking at this code, also add some note in a comment along the lines of https://github.com/OrchardCMS/OrchardCore/pull/16130#discussion_r1614033523 here.

Reply:
I don't think a comment is needed. We have an Obsolete warning that would help with the transmission.

Reply:
I think it adds absolutely 0 value from maintenance and add just noise. I'll add it just because you want to.

Reply:
Why I'm asking for such a comment is because I was wondering exactly like this, until I saw your comment under the PR. It's clear for you because you've just written this code, but a year later it might not be, and it's especially not clear for anybody else.

Reply:
It's a permission instance not a complex implementation. The permission was moved and an obsolete attribute was added. I doubt this comment will help any one. But regardless it's added